plugins/config-registry.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: config-registry
+spec:
+  homepage: https://github.com/mumoshu/kubeconf
+  shortDescription: Switch between your kubeconfigs
+  version: v0.2.0
+  description: |
+    Also known as "kubeconf", a utility to switch between kubeconfigs stored in your local filesystem.
+    Similar to "ctx" and "ns", but for switching the whole kubeconfig.
```

Comment:
Please wrap around at ~80 chars, so that it prints nicely on wide and narrow terminals.

I'd recommend to describe the workflow here, so that users understand what is happening:

1) init sets up the registry at `~/.kube/registry` where all registered kubeconfigs will be stored.
2) switch between registered kubeconfigs
3) register additional kubeconfig files.